kube-metrics-adapter.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: kube-metrics-adapter
+  version: 0.2.3
+  epoch: 0
+  description: General purpose metrics adapter for Kubernetes HPA metrics
+  copyright:
+    - license: MIT
+
+environment:
```

Comment:
We should be able to drop the `environment` definition here. `go/build` should retrieve everything we need :)